Sources/App/Views/PublicPage.swift

```diff
@@ -336,6 +336,12 @@ class PublicPage {
                                 "Privacy and Cookies"
                             )
                         ),
+                        .li(
+                            .a(
+                                .href("https://swiftpackageindex.statuspage.io"),
+                                "Uptime and System Status"
+                            )
+                        ),
                         .li(
```

Comment:
Excuse the noise in this diff, I found some indentation issues and so allowed Xcode to re-indent the whole file. This is the important bit of this diff, though!